projects/plugins/jetpack/extensions/blocks/subscriptions/menu.js

```diff
@@ -47,12 +47,12 @@ const NewsletterMenu = ( { openPreviewModal } ) => {
 			className="jetpack-newsletter-settings-sidebar"
 		>
 			<PanelBody>
-				{ ! isPublished && <NewsletterEmailDocumentSettings /> }
-				<SubscribersAffirmation accessLevel={ accessLevel } prePublish={ ! isPublished } />
 				{ isSendEmailEnabled && ! isPublished && (
 					<>
 						{ ! shouldPromptForConnection ? (
 							<>
+								{ ! isPublished && <NewsletterEmailDocumentSettings /> }
```

Comment:
This is within the `isSendEmailEnabled` condition, which is causing the toggle to disappear when emails are toggled off:

Before | After
---- | ----
<img width="299" alt="Screenshot 2025-06-03 at 1 45 24 PM" src="https://github.com/user-attachments/assets/8f46eed0-7491-439f-9e4e-25cf3a078a4e" /> | <img width="382" alt="Screenshot 2025-06-03 at 1 29 08 PM" src="https://github.com/user-attachments/assets/a10d4da5-f321-4568-82d2-590f3d327d9e" />

We could move the `isSendEmailEnabled` condition below it to wrap the preview and test email buttons.